events/app.py

```diff
@@ -72,7 +72,31 @@ def edit_event(event_id):
 @app.route('/v1/', methods=['GET'])
 def get_all_events():
     """Return a list of all events currently in the DB."""
-    pass
+    try:
+        events = events_collection.find()
+        events_dict = build_events_dict(events)
+        # TODO(cmei4444): test with pageserve to make sure the json format is
+        # correct in the response
+        return Response(
+            events_dict,
+            status=200,
+            mimetype='application/json'
+        )
```

Comment:
Instead of importing `json` and manually constructing a response, you can import `jsonify` from Flask.
```python
return jsonify(events_dict)
```
Status code 200 is default.